### ISSUE TITLE
User Story: Understand Testing Results and Trends

### ISSUE THREAD

**Issue by jmandel** [AudaciousInquiry/saner-ig#26]
Scenario: Alerting

As a [Public Health Official](../wiki/Persona#public-health-official) or [Treating Provider](../wiki/Persona#treating-provider)
I want to understand trends in a variety of metrics, e.g. as being developed [by Epic](https://chat.fhir.org/#narrow/stream/226195-Covid-19-Response/topic/Detailed.20definition.20of.20metrics):
So that I can be alerted when a facility or region surpasses established limits or matches the criteria. These can involve reporting numerators, denominators, and percentages:

![image](https://user-images.githubusercontent.com/313089/77677002-16992b80-6f5d-11ea-920c-a51aff65c699.png)

Given a date range and region, I should be able to look up these metrics and there component parts for any health systems in the region.

**Comment by michaeldonnelly** [comment]
The component parts are key.  We want a consumer to know not just what percent of encounters were with patients with a presumptive positive test; they should know the numerator and denominator in case (for example) they want to aggregate those with comparable metrics from other hospitals.

**Comment by keithboone** [comment]
Yeah, we can do just about anything with measures (I worked on Query Health and HQMF for a couple of years).

I'm focused on Utilization right now.  Surveillance is a separate set of issues with different data sources necessarily than utilization.  I know Epic can do both, but the world is not all about Epic (and don't get me wrong, I looked for a provider who uses Epic b/c MyChart is best available EHR-based patient portal from my patient perspective).  Some hospitals in my area use Meditech, others other systems, and they can't all report measures on surveillance data as readily.

Measurement is great, but to get to a measure, you have to get to the data behind it first, then aggregate, then count, then compute.  The same count can go into multiple measures.

**Comment by jmandel** [comment]
> Measurement is great, but to get to a measure, you have to get to the data behind it first, then aggregate, then count, then compute. The same count can go into multiple measures.

Is this story different for Groups? If you limit the scope of the IG to simple counts, then it's "easy" for Groups and MeasureReports; if you also have a need to support measures with numerators and denominators, then it's at least *possible* for MeasureReports.
